### DISCUSSION TITLE
PHP deprecations are logged

### DISCUSSION THREAD

**Discussion by eekes** [getsentry/sentry-symfony#591] [Q&A] [answered]
Hi guys!

I'm going crazy over a deprecation error that is logged thousands of times and I can't seem to suppress it.

This line:
https://github.com/sulu/sulu/blob/2.4.0/src/Sulu/Component/Webspace/Analyzer/Attributes/WebsiteRequestProcessor.php#L64

Causes this error to be logged in Sentry:
```
Deprecated: usort(): Returning bool from comparison function is deprecated, return an integer less than, equal to, or greater than zero
```

Any idea why this is happening and how I can suppress it? 😄

**Comment by stayallive** [comment]
@eekes, what have you tried so far?

Also, are you looking to silence only this one or all deprecations?

**Comment by glennmcewan** [comment, marked as answer]
As the error describes, the `usort` callback is returning booleans rather than integers which the function expects. Have a read through the function's documentation on php.net: https://www.php.net/manual/en/function.usort.php

This error seems to be an issue with the `sulu/sulu` repository; unrelated to Sentry or Symfony I think.
